packages/next/src/build/output/format.ts

```diff
@@ -0,0 +1,55 @@
+import type { CacheControl } from '../../server/lib/cache-control'
+
+const timeUnits = [
+  { label: 'y', seconds: 31536000 },
+  { label: 'w', seconds: 604800 },
+  { label: 'd', seconds: 86400 },
+  { label: 'h', seconds: 3600 },
+  { label: 'min', seconds: 60 },
+  { label: 's', seconds: 1 },
```

Comment:
This is mixing narrow and short format when we'd consider a standard `Intl.DurationFormat`

`short` would be
```suggestion
  { label: 'yr', seconds: 31536000 },
  { label: 'wk', seconds: 604800 },
  { label: 'days', seconds: 86400 },
  { label: 'hr', seconds: 3600 },
  { label: 'min', seconds: 60 },
  { label: 'sec', seconds: 1 },
```

`narrow` would be
```suggestion
  { label: 'y', seconds: 31536000 },
  { label: 'w', seconds: 604800 },
  { label: 'd', seconds: 86400 },
  { label: 'h', seconds: 3600 },
  { label: 'm', seconds: 60 },
  { label: 's', seconds: 1 },
```